framework/src/modules/chain/api/controllers/accounts.js

```diff
@@ -45,11 +45,10 @@ function accountFormatter(totalSupply, account) {
 		'address',
 		'publicKey',
 		'balance',
-		'u_balance',
 		'secondPublicKey',
 	]);
-	object.unconfirmedBalance = object.u_balance;
```

Comment:
The API will expose the unconfirmedBalance property because this feature is developed in a minor release. Therefore, the unconfirmedBalance should equal the confirmed balance (which will usually be the case when the account does not have any transactions in the pool)

Reply:
Fixed.